lib/stripe/profile_storer.rb

```diff
@@ -67,11 +67,7 @@ def customer_profile_id(response)
     end
 
     def payment_profile_id(response)
-      if response.params['customer'] # Payment Intents API
-        response.params['id']
-      else
-        response.params['default_source'] || response.params['default_card']
-      end
```

Comment:
Based on the comment, I would have expected that we should now return `response.params['id']` since our SCA gateway uses the Payment Intents API.

Reply:
Ah disregard this comment, I was going through each commit. Looks like you changed it back.